tcp_server/Containerfile

```diff
@@ -4,7 +4,7 @@ RUN apk add \
 	make \
 	;
 
-ADD . /tcp_server
```

Comment:
Makes sense, but what features does ADD have over COPY?

Reply:
ADD can take a url or a local path (instead of only local path like COPY) and if the remote/local file is an archive (e.g. tar.gz) it will automatically extract its contents (instead of copying the archive file itself like COPY) into the destination within the container.